code/socialDistribution/views.py

```diff
@@ -155,7 +151,8 @@ def posts(request, author_id):
         count = 0
 
         try:
-            post = Post.objects.create(
+            if img:
```

Comment:
A bit picky, but can we fix the indentation here to make it consistent? Also, any way we could refactor this code to make it a bit shorter? Just feels like the function is getting quite long. Open to suggestions, but we can just leave it as-is for now